srcpkgs/uhubctl/template

```diff
@@ -0,0 +1,14 @@
+# Template file for 'uhubctl'
+pkgname=uhubctl
+version=2.0.0
+revision=1
+build_style=gnu-makefile
+make_use_env=yes
+makedepends="libusb-devel"
+depends="libusb"
```

Comment:
This `depends` line is unnecessary because xbps-src detects this automatically.

https://travis-ci.org/void-linux/void-packages/jobs/570475910#L4050

Reply:
Fixed.

Even though the manual [describes](https://github.com/void-linux/void-packages/blob/master/Manual.md#deps_runtime) that feature, it would be great to catch redundant dependencies automatically as well (maybe, by printing a warning in a CI build?).

BTW, thanks for your great work! Void Linux is amazing.